index.html

```diff
@@ -12,24 +12,23 @@
     <title>Danicas Website</title>
   </head>
   <body>
-    <header id="top">
-      <h1>Danica Dombek</h1>
-      <nav>
-        <a href="#biographie" class="navi-buttons">Biographie</a>
-        <a href="#kontakt" class="navi-buttons">Kontakt</a>
+    <header id="top" class="header">
+      <h1 class="header__h1">Danica Dombek</h1>
+      <nav class="header__nav">
+        <a href="#biographie" class="header__navi-buttons">Biographie</a>
+        <a href="#kontakt" class="header__navi-buttons">Kontakt</a>
       </nav>
     </section>
     </header>
 
-    <main>
-      
-        <img class="image-flex" src="images/dd-site.jpg" alt=""/>
+    <main>   
+        <img class="img-fluid" src="images/dd-site.jpg" alt=""/>
         <figcaption>"Das Leben ist kurz." Seneca</figcaption>
         
-        <section class="columns standard-margin">   
+        <section class="skills skills__m-20 skills__columns">   
         <div>
-          <h2>Fähigkeiten</h2>
-          <ul> 
+          <h2 class="skills__h2">Fähigkeiten</h2>
```

Comment:
Man vermeidet meistens die Tag-namen in BEM zu benutzen sprich skills__h2 wäre dann eher skills__title bspw.

Reply:
Another way to name it:
```suggestion
          <h2 class="skills__heading">Fähigkeiten</h2>
```